core/domain/exp_services.py

```diff
@@ -1010,9 +1020,8 @@ def _create_exploration(
             ).to_dict()
             for state_name in exploration.states
         },
-        metadata_version_history=(
-            exp_domain.ExplorationMetadataVersionHistory(
-                None, committer_id).to_dict()),
+        metadata_last_edited_version_number=0,
```

Comment:
Yes, it should have been None similar to what we do for the states. Earlier, I made the property `metadata_last_edited_version_number` as required, hence, it was showing errors.

I have changed this now to a not-required property and this can be None now.